content/components/buttons/examples/example-default/example-code.md

```diff
@@ -1 +0,0 @@
-<button class="au-btn">Primary button</button>
```

Comment:
Can we delete this file if we are only using `example-code.md` and `example-dark.md` from `content/components/buttons/examples/example-widths`